member/models.py

```diff
@@ -164,3 +165,23 @@ class MemberPreferences(models.Model):
     show_long_agenda = models.BooleanField(default=True)
 
     default_view = models.IntegerField(choices=AgendaChoices.choices, default=AgendaChoices.AGENDA)
+
+
+class Invite(models.Model):
+    """
+    An invitation sent to an email address.  The recipient can sign up with that or
+    another email address.
+    """
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    band = models.ForeignKey(Band, related_name="invites", on_delete=models.CASCADE, null=True)
+    email = models.EmailField(_('email address'))
+    language = models.CharField(choices=LANGUAGES, max_length=200, default='en')
+
+    @property
+    def email_line(self):
+        return self.email
+
+    # We want language to work as it does for members
+    @property
+    def preferences(self):
+        return SimpleNamespace(language=self.language)
```

Comment:
I don't understand - what's the thinking with this?

Reply:
I wanted to be able to reuse member.helpers.prepare_email with Invite objects, since that takes care of translation and HTML formatting.  This function is expecting Member objects, but the only part of that API I was using was the `.email_line` and `.preferences.language` attributes.  This code duplicates those attributes in the Invite object, so it can be used here as well.

With some distance, I don't particularly like this solution.  Here's two other ideas:
1. Add a `.language` property to Members, which looks up the language in their preferences.  Then, `prepare_email` can use that API, and we don't have to jump through hoops here.
2. Create a `EmailRecipient` class / dictionary / named tuple that `prepare_email` will accept, and then build a way for both Member and Invite objects to create such objects.  This would keep us from accidentally using properties of one that the other doesn't have.

Any thoughts?